README.md

```diff
@@ -7,11 +7,39 @@ Build tools for the Seam API using this blueprint.
 
 ## Description
 
-TODO
+A blueprint is a simplified description of an API schema that is optimized for code and documentation generation.
+
+The blueprint schema assumes the API follows Seam API design guidelines,
+and will evolve with those guidelines and any SDK, documentation, or product quality requirements.
+
+The blueprint defines the following core concepts:
+
+- Endpoint: a path that accepts an HTTP request and returns a response.
+- Route: the direct parent path of a collection of one or more endpoints.
+- Subroute: A route that is nested under another route.
+- Namespace: the direct parent path of a collection routes only: there are no endpoints under this path.
```

Comment:
```suggestion
- Namespace: the direct parent path of a collection of routes only: there are no endpoints under this path.
```

Reply:
> there are no endpoints under this path.
This line also makes me think that not only `acs` is a namespace, as we discussed in the docs meeting, but `noise_sensors` might be as well. Although, `/noise_sensors` could potentially have `/get` and `/list` endpoints since we have those for `/locks`.

Reply:
> there are no endpoints under this path.

Does it mean that `noise_sensors` is also a namespace, besides `acs`? Although, `/noise_sensors` could potentially have `/get` and `/list` endpoints since we have those for `/locks`.

Reply:
`/noise_sensors/list` should exist if it doesn't, it is not intended to be a namespace.

Reply:
There are no endpoints under `/nosie_sensors`. What about `/noise_sensors/get`, should we add it as well?

Reply:
Actually, only `/nosie_sensors/list`

Reply:
Created an issue https://github.com/seamapi/seam-connect/issues/7520